### PR TITLE
Feature admin application methods

### PR DESCRIPTION
**Pull request by varffs**
delete, mark paid and unsubmit

### CODE REVIEW

File: app/imports/api/methods/adminApplicationMethods.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  },
+
+  run(applicationId) {
+    if (!Roles.userIsInRole(this.userId, 'admin')) {
+      throw new Meteor.Error('Admin.methods.add-user.not-allowed', 'Must be admin to do this.');
+    }
+
+    let application = Applications.findOne(applicationId);
+
+    Applications.update(application._id, {
```
Comment (mixcas) [inline app/imports/api/methods/adminApplicationMethods.js]: Line 58 is redundant to 60 no? cuz `applicationId == application._id`. There's no point on looking for it first.

Comment (mixcas) [inline app/imports/api/methods/adminApplicationMethods.js]: Line 58 is redundant to 60 no? cuz `applicationId == application._id`. There's no point on looking for it first.

File: app/imports/api/methods/adminApplicationMethods.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
```
Comment (mixcas) [inline app/imports/api/methods/adminApplicationMethods.js]: Same here

File: app/imports/api/methods/adminApplicationMethods.js (hunk 1 above)
Comment (varffs) [inline app/imports/api/methods/adminApplicationMethods.js]: to verify that it is a real application? but yh maybe true prefer that the method is called with a real application and anyhow I never check to see if there is a return value

File: app/imports/components/applications/applicationsLayout.jsx [hunk 3]
```diff
@@ -60,6 +64,57 @@ export class ApplicationsLayout extends Component {
 }
 
 export class ApplicationsApplication extends Component {
```
Comment (mixcas) [inline app/imports/components/applications/applicationsLayout.jsx:66]: This naming is confusin

Comment (varffs) [inline app/imports/components/applications/applicationsLayout.jsx:66]: its not new code

Comment (varffs) [inline app/imports/components/applications/applicationsLayout.jsx:66]: but yh it isnt good code either

File: app/imports/api/methods/adminApplicationMethods.js (hunk 1 above)
Comment (mixcas) [inline app/imports/api/methods/adminApplicationMethods.js]: Update runs a find for itself and would error out the same if non-existant applicationId is passed.

Comment (varffs) [inline app/imports/api/methods/adminApplicationMethods.js]: that error would never be returned in any way tho unless checked for

Comment (mixcas) [inline app/imports/api/methods/adminApplicationMethods.js]: Yeah, we should catch the return of the update and return proper `message` depending if err or success

Comment (varffs) [inline app/imports/api/methods/adminApplicationMethods.js]: That works for everything except delete tho

**Pull request merged**